### ISSUE TITLE
GUI Brain Atlas: Clear selection from Menu not working 

### ISSUE THREAD

**Issue by annacanal** [softmatterlab/Braph-2.0-Matlab#431]
When pressing: Clear selection in the Menu, it gives an error
![image](https://user-images.githubusercontent.com/31701548/85301022-9aed4300-b4a7-11ea-9f9d-48dd96b935e9.png)

Error:
```
Undefined function 'cg_table_clearselection' for input arguments of type 'matlab.ui.container.Menu'.
Error while evaluating Menu Callback.
```

When clearing the selection by pressing the bottom button, then it works.

**Comment by annacanal** [comment]
Similarly, move to top fails when pressed in the menu
![image](https://user-images.githubusercontent.com/31701548/85302356-52cf2000-b4a9-11ea-88b7-5af14da6c0eb.png)
giving the error:

```
Undefined function 'cb_vmove2top' for input arguments of type 'matlab.ui.container.Menu'.
Error while evaluating Menu Callback.
```

but works when pressing the button above 
![image](https://user-images.githubusercontent.com/31701548/85302286-4054e680-b4a9-11ea-8062-33841ce9fc20.png)

**Comment by egolol** [comment]
solved in #437
